CHANGELOG.rst

```diff
@@ -1,40 +1,49 @@
 2.9.0.dev
 =========
 
-New Features
-------------
+**New Features**
```

Comment:
> I'm guessing rst-lint exits non-zero in this case (with only INFO messages) as well? 

Yes, even with only `INFO` messages it would return 1. :grimacing: